src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -431,3 +442,123 @@ class Event(NamedTuple):
 
 
 Section = List[Event]
+
+# --- System Logs ---
+
+
+class SystemLogSourceType(str, Enum):
+    """Types of sources that can emit system logs."""
+
+    RAY_HEAD_NODE = "RAY_HEAD_NODE"
+    RAY_WORKER_NODE = "RAY_WORKER_NODE"
+    K8S_EVENT = "K8S_EVENT"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Should we include an "unknown" source type that uses the log as a string?

Reply:
Good idea, I've implemented an UNKNOWN value for the enum with a corresponding event log :)